OnBoarding/OnBoarding.php

```diff
@@ -182,14 +164,14 @@ private function loadSteps($configPath, $languageIsoCode)
      */
     private function getTextFromSettings($text)
     {
-        switch (array_keys($text)[0]) {
-            case 'loc':
-                return $this->localization[$text['loc']];
-            case 'content':
-                return $this->getTemplateContent('contents/'.$text['content']);
+        if (is_array($text)) {
+            switch ($text['type']) {
```

Comment:
We need to always have $text['type'], maybe one array_key_exist ? 
(if someday, someone change the config not formatting like you.. slash)